Marlin/src/core/language.h

```diff
@@ -471,6 +468,10 @@
   #define AXIS6_STR   ""
 #endif
 
+#define STR_I AXIS4_STR
+#define STR_J AXIS5_STR
+#define STR_K AXIS6_STR
```

Comment:
Macros expand on usage, not on define, so it is fine for these to be above the definitions of `AXIS#_STR`.